extension/app/ts/background/popupMessageHandlers.ts

```diff
@@ -211,3 +211,20 @@ export async function getAddressBookData(_simulator: Simulator, payload: PopupMe
 		}
 	})
 }
+
+export async function openAddressBook(_simulator: Simulator, _payload: PopupMessage) {
+	const html = '/html/addressBook.html'
+
+	const openInNewTab = async () => {
+		const tab = await browser.tabs.create({ url: html })
```

Comment:
I think this is what @0xjimmy was getting at previously.  The local variable doesn't provide any value here since this is only used once.  Might as well just inline it to avoid the reader needing to do a lookup.
```suggestion
	const openInNewTab = async () => {
		const tab = await browser.tabs.create({ url: '/html/addressBook.html' })
```